datasets/nell/nell.py

```diff
@@ -0,0 +1,218 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NELL: Never Ending Language Learner"""
+
+from __future__ import absolute_import, division, print_function
+
+from html import unescape
+from urllib.parse import unquote
+
+import datasets
+
+
+_CITATION = """\
+Never-Ending Learning.
+T. Mitchell, W. Cohen, E. Hruschka, P. Talukdar, J. Betteridge, A. Carlson, B. Dalvi, M. Gardner, B. Kisiel, J. Krishnamurthy, N. Lao, K. Mazaitis, T. Mohamed, N. Nakashole, E. Platanios, A. Ritter, M. Samadi, B. Settles, R. Wang, D. Wijaya, A. Gupta, X. Chen, A. Saparov, M. Greaves, J. Welling. In Proceedings of the Conference on Artificial Intelligence (AAAI), 2015
+"""
+
+
+_DESCRIPTION = """This dataset provides version 1115 of the belief
+extracted by CMU's Never Ending Language Learner (NELL) and version
+1110 of the candidate belief extracted by NELL. See
+http://rtw.ml.cmu.edu/rtw/overview.  NELL is an open information
+extraction system that attempts to read the Clueweb09 of 500 million
+web pages (http://boston.lti.cs.cmu.edu/Data/clueweb09/) and general
+web searches.
+
+The dataset has 4 configurations: nell_belief, nell_candidate,
+nell_belief_sentences, and nell_candidate_sentences. nell_belief is
+certainties of belief are lower. The two sentences config extracts the
+CPL sentence patterns filled with the applicable 'best' literal string
+for the entities filled into the sentence patterns. And also provides
+sentences found using web searches containing the entities and
+relationships.
+
+There are roughly 21M entries for nell_belief_sentences, and 100M
+sentences for nell_candidate_sentences.
+"""
+
+
+_LICENSE = """There does not appear to be a license for this dataset. It is made available by Carnegie Mellon University at http://rtw.ml.cmu.edu/rtw/resources
```

Comment:
```suggestion
_LICENSE = """
```
Can you mention that in the dataset card instead ?
Here we expect the name or description of the license if there's any.

Reply:
Done